.azure-pipelines/build-template.yml

```diff
@@ -41,6 +41,10 @@ jobs:
 - job:
   displayName: ${{ parameters.arch }}
   timeoutInMinutes: ${{ parameters.timeout }}
+  variables:
+    DIFF_COVER_CHECK_THRESHOLD: 0
```

Comment:
Do you have some document about the two variables? What is the value range, and how do we choose the value?

Reply:
@qiluo-msft , add the doc https://github.com/Azure/sonic-build-web/tree/main/code-diff-coverage